met-api/src/met_api/constants/timeline_event_status.py

```diff
@@ -0,0 +1,23 @@
+# Copyright © 2023 Province of British Columbia
```

Comment:
I think we should do away with this open source license boilerplate for files. I don't think it adds anymore than having the open source license declaration in the root of the project. I recognize that you used other files as "templates" for your work by copy-pasting but this is one thing I don't think needs to come over.